src/aggregate.h

```diff
@@ -273,7 +273,7 @@ class ClassDeclaration : public AggregateDeclaration
     bool com;                           // true if this is a COM class (meaning it derives from IUnknown)
     bool cpp;                           // true if this is a C++ interface
     bool isscope;                       // true if this is a scope class
-    bool isabstract;                    // true if abstract class
+    int isabstract;                     // 0: fwdref, 1: is abstract class, 2: not abstract
```

Comment:
Also, `isabstract` becomes a bad name when it is tri-state, because `if (isabstract)` will be true even if `isabstract` is 2 !